oeps/architectural-decisions/oep-0041-arch-async-server-event-messaging.rst

```diff
@@ -475,10 +475,16 @@ Having a clearly defined set of events would allow for simpler third party
 integration in areas like student learner event processing and course catalog
 management.
 
+------------
+
+.. note:: All future decisions related to :doc:`Open edX Events <openedx-events:index>` that adhere to these guidelines for asynchronous event messaging will now be documented and made publicly available through the Architectural Decision Records (ADRs) in :doc:`Open edX Events ADRs <openedx-events:decisions/index>`. This includes decisions regarding the Event Bus, schema format, serialization, evolution, and design practices for event-driven architecture.
```

Comment:
Reading this gave me additional ideas that you may like or not like.

1. This reads like a small decision. I’m wondering if there could be a brief OEP that describes this decision  named “Event Related Decision Records”, or “Event and Filter…” or “Hook, Event and Filter…” that details this change, and notes could point to this OEP?
2. Maybe this won’t be the last time we choose a library repo for housing OEP-esque ADRs on a topic? We could consider updating OEP-19 with this as an option, or an OEP-19 sub ADR, and also list all the current cases as an index.

These are just ideas, and they could influence how we add the notes. That said, we could make these notes the first iteration.

Reply:
I disagree with this - I think that's adding too much process that no one will ultimately end up reading/needing. I think this note in the OEP is enough. I ran this by Feanil and Kyle and they agree with me, as well.

Reply:
Understood. It’s really all just a guess about how people will find the docs they are looking for, and the agree the notes should be sufficient for the problem at hand.